src/Microsoft.AspNet.Razor/Properties/AssemblyInfo.cs

```diff
@@ -5,5 +5,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Test")]
-[assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Runtime.Test")]
```

Comment:
Nothing new: https://github.com/aspnet/Home/wiki/Engineering-guidelines#when-to-use-internals-vs-public-and-when-to-use-internalsvisibleto

This seems like an opportunistic fix.